cmd/helm/lint.go

```diff
@@ -51,19 +51,33 @@ func init() {
 	RootCommand.AddCommand(lintCommand)
 }
 
-var errLintNoChart = errors.New("no chart found for linting (missing Chart.yaml)")
-var errLintFailed = errors.New("lint failed")
+var errLintNoChart = errors.New("No chart found for linting (missing Chart.yaml)")
```

Comment:
`gofmt` seems fine, `golint` reported a few issues though so I'll go through and fix those. Though, this line in particular didn't have any issues - are you seeing something different?